org.scala-ide.sdt.core.tests/src/org/scalaide/core/sbtbuilder/ProjectDependenciesTest.scala

```diff
@@ -98,6 +99,7 @@ class ProjectDependenciesTest {
     }
   }
 
+  @Ignore
```

Comment:
I meant "on Scala's Jekins".

It simply passes on Scala IDE's Jenkins and when someone runs tests on his/her computer.

Except that I don't know. // I'm just reading these comments since I'm getting notifications (not that I was investigating this issue or something like that).